assembly/src/parsers/mod.rs

```diff
@@ -323,14 +326,25 @@ fn parse_param<I: core::str::FromStr>(op: &Token, param_idx: usize) -> Result<I,
     Ok(result)
 }
 
+/// A helper function that specifies if the Range Query is inclusive or exclusive.
+fn bound_into_included_u64<I>(bound: Bound<&I>) -> u64
```

Comment:
optional nit: might be overkill to promote this function to the root module since it is used both in `parsers` and `instruction`, but on the other hand we should avoid repeating code.

Reply:
I moved it to utils. That's what I asked earlier. If we should move this function somewhere else to be easily accessed across different modules.

Reply:
Yes, that would be the right choice as we call it in multiple places